jsdrupal/drupal-admin-ui
# unit test: testResultsProcessor is deprecated use reporters #626

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/626

## Testing instructions

As per the issue. run unit test with

yarn workspace admin-ui test:unit

and observe that the deprecation warnings are missing.
